cloud9_config.py

```diff
@@ -0,0 +1,138 @@
+""" Script to simplify the Cloud9 IDE configuration. 
+This works together with the CloudFormation template used to seed the workshop and a `roboMakerSettings.json` configuration file supplied
+as part of this project. Any changes to those will require reviewing this script
+"""
+
+import os
+import json
+import argparse
+import re
+import shutil
+
+import boto3
+
+# LAUNCH_FILE_PATH = 'robot_ws/src/martian_detector/launch/martian_detector.launch'
+CONFIG_FILE_NAME = 'roboMakerSettings.json'
+SIM_CONFIG_ID = 'sim-robot-with-simple-world'
+BUCKET_KEY = 'RoboMakerS3Bucket'
+REGION_KEY = 'REGION'
+CFN_CONFIG_KEYS = {
+    BUCKET_KEY: ['cfg.simulation.outputLocation', 'cfg.robotApp.s3Bucket', 'cfg.simulationApp.s3Bucket', 'cfg.robotApp.launchConfig.environmentVariables.S3_BUCKET'],
```

Comment:
There is a BUCKET_KEY variable already declared a bit above this. Can lead to confusion.